packages/standalone/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@typespec/standalone-cli",
   "private": true,
-  "version": "0.67.3",
+  "version": "0.68.0",
```

Comment:
Is this going to cause any funny issues with how versioning is displayed by the CLI? Or is it ok for it to show `0.68.0`?

Reply:
its going to be a tiny bit weird that it downloads version 0.68 but it will still show the version of teh compiler installed that version it kinda hidden so don't think its a big deal for now